src/main/scala/codacy/checkstyle/DocGenerator.scala

```diff
@@ -150,4 +150,24 @@ object DocGenerator {
     }
   }
 
+
+  implicit class StringOps(val s: String) extends AnyVal {
```

Comment:
`StringOps` in Scala std lib is already pretty comprehensive, are we sure we can't avoid running our own implementation of StringOps? e.g. `takeWhile` have the same signature and in case a `reverse` will make the trick of removing from the other end ...